src/nsls2api/services/proposal_service.py

```diff
@@ -336,18 +426,12 @@ async def has_valid_cycle(proposal: Proposal):
     # proposal then the cycle information is invalid
     return not (
         (len(proposal.cycles) == 0)
-        and (
-            proposal.pass_type_id != "300005"
-            or proposal.pass_type_id != "300042"
-        )
+        and (proposal.pass_type_id != "300005" or proposal.pass_type_id != "300042")
     )
 
 
 async def is_commissioning(proposal: Proposal):
-    return (
-        proposal.pass_type_id == "300005"
-        or proposal.pass_type_id == "300042"
-    )
+    return proposal.pass_type_id == "300005" or proposal.pass_type_id == "300042"
```

Comment:
Could be clearer if using `commissioning_proposal_types` as defined above, rather than magic values here.

```suggestion
    return proposal.pass_type_id in commissioning_proposal_types
```

Reply:
I've added a helper function in `pass_services` to return the list of commissioning proposal type IDs and then reusing it in the proposal_service.

Thanks for the suggestion and prompting me to fix some of my TODOs 😄